test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -230,7 +254,16 @@ void ProtoConverter::visit(Expression const& _x)
 
 void ProtoConverter::visit(BinaryOp const& _x)
 {
-	switch (_x.op())
+	BinaryOp_BOp op = _x.op();
+
+	if ((op == BinaryOp::SHL || op == BinaryOp::SHR || op == BinaryOp::SAR) &&
```

Comment:
Maybe add a temporary variable `isShiftOperation` and then have the condition only in one line?